.ci.yaml

```diff
@@ -1232,6 +1232,15 @@ targets:
       - bin/**
       - .ci.yaml
 
+  - name: Linux_android_emu flutter_driver_android_test
+    recipe: devicelab/devicelab_drone
```

Comment:
I think the devicelab_drone recipe invokes the //dev/devicelab/bin/test_runner.dart entrypoint: https://flutter.googlesource.com/recipes/+/refs/heads/main/recipes/devicelab/devicelab_drone.py#44 and not the test.dart one. I think you will need to either:

1. make your test a devicelab task; or
2. make this target use the flutter/flutter_drone recipe, and set the `shard` property field, which is what `test.dart` uses for the key in that hash map you updated.

Reply:
I ... see. Let me try the latter and see what happens I guess.